Auth0/Auth0WebAuth.swift

```diff
@@ -199,24 +187,19 @@ final class Auth0WebAuth: WebAuth {
         var components = URLComponents(url: authorize, resolvingAgainstBaseURL: true)!
         var items: [URLQueryItem] = []
         var entries = defaults
+
         entries["client_id"] = self.clientId
         entries["redirect_uri"] = redirectURL.absoluteString
+        entries["response_type"] = responseType
         entries["scope"] = requiredScope // TODO: Change when setting the new default scope
         entries["state"] = state
-        entries["response_type"] = self.responseType.map { $0.label! }.joined(separator: " ")
+        entries["nonce"] = nonce
+        entries["organization"] = organization
+        entries["invitation"] = invitation
```

Comment:
It's not really necessary to have `if let organization = organization`... to add optional values (value that may be `nil`). If it's `nil`, Swift won't add anything to the dictionary.
It's only necessary in the case of `maxAge` because it needs to be converted to String first, and for that, it must not be optional.

Reply:
See https://forums.swift.org/t/dictionary-with-optional-values/2675/8